docs/localdatetime.md

```diff
@@ -0,0 +1,1242 @@
+# Temporal.LocalDateTime
+
+<details>
+  <summary><strong>Table of Contents</strong></summary>
+<!-- toc -->
+</details>
+
+> **NOTE**: This type has been approved and will be merged soon (probably early October 2020), but it's not in current builds yet.
+> Also, "LocalDateTime" is a placeholder name. Its final name is being discussed [here](https://github.com/tc39/proposal-temporal/issues/707).
+> If you have feedback about this new type, visit https://github.com/tc39/proposal-temporal/issues/700.
+
+A `Temporal.LocalDateTime` is a time-zone-aware, calendar-aware date/time type that represents a real event that has happened (or will happen) at a particular instant in a real place on Earth.
+As the most comprehensive `Temporal` type, `Temporal.LocalDateTime` can be considered a combination of `Temporal.TimeZone`, `Temporal.Absolute`, and `Temporal.DateTime` (which includes `Temporal.Calendar`).
```

Comment:
"The most information" seems kind of vague-- I think the connotation we want is expansive or thorough, not just "more" or "most". Got any other idea for how to describe it?  "The most complete"?  Something else?

Reply:
Comprehensive is not bad in that sense, the reason I suggested using another word is to avoid the connotation of "you should always use LocalDateTime for everything." Maybe "most specific"?

Reply:
I'm not a fan of "most specific" because if you're thinking that "specific" refers to use cases, then a type like MonthDay is "most specific" because it targets narrower use cases. But if you're thinking "specific" means data, then LDT is the most specific. I think we should pick a different term to avoid this potential ambiguity.

I changed to "broadest" which seems to be the least judgmental word I could find for the concept. I'm happy to pick another term but if OK with you I'll resolve this now so we can get the docs in and we can iterate. Sound OK?